ground/src/main/java/com/google/android/ground/GroundApplication.kt

```diff
@@ -34,14 +34,11 @@ class GroundApplication : MultiDexApplication(), Configuration.Provider {
   @Inject lateinit var crashReportingTree: CrashReportingTree
   @Inject lateinit var workerFactory: HiltWorkerFactory
 
-  init {
-    Timber.plant(if (isReleaseBuild()) crashReportingTree else Timber.DebugTree())
-  }
-
   private fun isReleaseBuild(): Boolean = BuildConfig.BUILD_TYPE.contentEquals("release")
 
   override fun onCreate() {
     super.onCreate()
+    Timber.plant(if (isReleaseBuild()) crashReportingTree else Timber.DebugTree())
```

Comment:
Ah, I see - it was _us_ calling `getCrashReportingTree()` in the constructor, not the Firebase SDK. Thanks for fixing this! @rfontanarosa FYI.